Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.messages.in

```diff
@@ -36,11 +36,15 @@ messages -> RemoteRenderingBackend NotRefCounted Stream {
     ReleaseAllImageResources()
     ReleaseRenderingResource(WebCore::RenderingResourceIdentifier renderingResourceIdentifier)
 
+    CreateRemoteRenderingBufferSet(WebKit::RemoteBufferSetIdentifier bufferSetIdentifier)
+    ReleaseRemoteRenderingBufferSet(WebKit::RemoteBufferSetIdentifier bufferSetIdentifier)
+    UpdateRemoteRenderingBufferSetConfiguration(WebKit::RemoteBufferSetIdentifier bufferSetIdentifier, WebCore::FloatSize logicalSize, WebCore::RenderingMode renderingMode, float resolutionScale, WebCore::DestinationColorSpace colorSpace, enum:uint8_t WebCore::PixelFormat pixelFormat)
+
 #if PLATFORM(COCOA)
-    PrepareBuffersForDisplay(Vector<WebKit::PrepareBackingStoreBuffersInputData> swapBuffersInput) -> (Vector<WebKit::PrepareBackingStoreBuffersOutputData> swapBuffersOutput) Synchronous NotStreamEncodableReply
+    CreateBuffersForDisplay(Vector<WebKit::PrepareBackingStoreBuffersInputData> swapBuffersInput)  -> (Vector<WebKit::PrepareBackingStoreBuffersOutputData> swapBuffersOutput) NotStreamEncodable NotStreamEncodableReply ReplyCanDispatchOutOfOrder
```

Comment:
What was the reason for NotStreamEncodable?
I'd think that's only if we *send* MachPorts, and I don't think swapBuffersInput has those?

Reply:
Looks like I was just failing to put the right annotations in the serialisation generator file, fixed now.